packages/core/src/axis-chart.ts

```diff
@@ -65,9 +66,30 @@ export class AxisChart extends Chart {
 			}
 		};
 
+		// Add extra legend items
+		const extraLabels = [];
+		const radiusLabel = Tools.getProperty(
+			options,
+			"bubble",
+			"radiusLabel"
+		);
+		if (radiusLabel) {
+			extraLabels.push({
+				type: "radius-label",
+				icon: `<svg width="16px" height="16px" viewBox="0 0 16 16" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+							<title>Artboard</title>
+							<g id="Artboard" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
+								<circle id="Oval-Copy-33" stroke="#8C8C8C" cx="7" cy="7" r="6.5"></circle>
+								<circle id="Oval-Copy-33" stroke="#8C8C8C" cx="7" cy="10" r="3.5"></circle>
+							</g>
+						</svg>`,
```

Comment:
can we clean this up a bit? 

1. <title> seems to have been carried over from the design file but its not being used correctly here 
2. do we need IDs on the <g> and <circle>? are we using the IDs? and if we are, im sure we could find better names 
3. can we remove some styles that are applied in attributes to the scss? especially the HEX color, stroke widths and stuff?